### PR TITLE
Add library and collection scripts

### PR DESCRIPTION
**Pull request by leonardr**
This branch adds four new scripts. One of them lists the libraries in the database, and one of them creates a new library or reconfigures an existing library.

The other two scripts do the same for collections.

The goal is to provide a simple way of doing initial configuration from the command line, and to provide a stopgap until the administrative interface allows the configuration of library and collection.

These scripts are in core rather than in circulation because I'm pretty sure the content server, at least, is going to use the concept of collection.

### CODE REVIEW

**Comment by aslagle** [comment]
Did you forget to add script files in bin or will those be in circulation?

**Comment by leonardr** [comment]
They'll be in circulation.

File: scripts.py [hunk 1]
```diff
@@ -541,6 +546,289 @@ def refresh_metadata(self, identifier):
...
+        if library_names:
+            return '"' + '", "'.join(library_names) + '"'
+        return ""
+    
+    def do_run(self, _db=None, cmd_args=None, output=sys.stdout):
+        _db = _db or self._db
+        args = self.parse_command_line(_db, cmd_args=cmd_args)
+
+        # Find or create the collection
+        create_kwargs = dict()
```
Comment (aslagle) [inline scripts.py]: you didn't end up using this?

File: tests/test_model.py [hunk 2]
```diff
@@ -5292,6 +5316,40 @@ def test_set_key_value_pair(self):
         eq_(setting2, collection.setting("website_id"))
 
 
+    def test_explain(self):
+        """Test that Library.explain gives all relevant information
+        about a Library.
```
Comment (aslagle) [inline tests/test_model.py]: This is Collection.explain

**Review by aslagle — APPROVED** [review]
Looks good minus an extraneous variable and an incorrect comment

**Comment by leonardr** [comment]
Travis integration is broken because of the S3 outage. I've run the test suite locally and verified it passes.

**Pull request merged**